### PR TITLE
Review documentation of Bifrost

### PR DESCRIPTION
**Pull request by darakeon**
Review documentation of Bifrost and move example-site to docs folder

### CODE REVIEW

File: docs/websocket.md [hunk 1]
```diff
@@ -1,162 +1,221 @@
...
 
 All messages are encoded in JSON. All request messages have the following parameter:**
 
-**context:** The [context](architecture.md#context).
+**contextId:** The id of [context](architecture.md#context). Need to be the same on a transaction processing.
 
-**command:** The command to be executed.
-
-**command_id:** Request ID, optional. Used to track the response.
+**requestType:** Step of the transaction processing.
```
Comment (guarilha) [inline docs/websocket.md]: @darakeon  This is really confusing. If the description is correct, I'd expect this parameter to be `transactionProcessingStep`. If the parameter name is correct, I'd expect an explanation that involves the request type.

File: docs/websocket.md [hunk 2]
```diff
@@ -1,162 +1,221 @@
...
 
-A websocket endpoint is provided on `/ws`.
+A websocket endpoint is provided on `/mpos`.
 
 # Protocol
 
 All messages are encoded in JSON. All request messages have the following parameter:**
 
-**context:** The [context](architecture.md#context).
+**contextId:** The id of [context](architecture.md#context). Need to be the same on a transaction processing.
```
Comment (guarilha) [inline docs/websocket.md]: This sentence "Need to be the same on a transaction processing" is not clear enough. How do I know the transaction processing? It's best if we assume people want more details here.

File: docs/websocket.md [hunk 3]
```diff
@@ -1,162 +1,221 @@
...
-**command_id:** Request ID, optional. Used to track the response.
+**requestType:** Step of the transaction processing.
 
 All responses contains the following parameters:**
 
-**command:** The command executed.
+**responseType:** Type of the response, which can be success response, error on process or unknown command, in case of request type unknown.
 
-**context:** The context where the command was executed
+**error:** Error message if any ocurred. Response Type, in this caso, will be 9.
```
Comment (guarilha) [inline docs/websocket.md]: The error message if any ocurred. `responseType` will be `9`.

File: docs/websocket.md [hunk 4]
```diff
@@ -1,162 +1,221 @@
...
 
-**command:** The command to be executed.
-
-**command_id:** Request ID, optional. Used to track the response.
+**requestType:** Step of the transaction processing.
 
 All responses contains the following parameters:**
 
-**command:** The command executed.
+**responseType:** Type of the response, which can be success response, error on process or unknown command, in case of request type unknown.
```
Comment (guarilha) [inline docs/websocket.md]: please add the list of possible values for `responseType` on a list such as: 
```
* `9`: an error has ocurred  
```

File: docs/websocket.md [hunk 5]
```diff
@@ -1,162 +1,221 @@
...
+**error:** Error message if any ocurred. Response Type, in this caso, will be 9.
 
-**command_id:** Echoes the value provided in the request.
+In this documentation only request/response examples will be provided. For information on actual parameters, check [this](command.md).
 
-**success:** Wheter the command was executed successfuly.
+## List Devices
 
-**response:** The actual response data, if the command has any.
+**RequestType:** "1" or "listDevices"
```
Comment (guarilha) [inline docs/websocket.md]: I'd change " for ` here. (and wherever we use " referring to values or parameters/responses strings)

File: docs/websocket.md (hunk 2 above)
Comment (darakeon) [inline docs/websocket.md]: I will remove the explanation, because there is a link to context documentation there.

**Comment by darakeon** [comment]
@guarilha, "Separate Config File" here and in another branch was a rebase accident actually. And this documentation is about Bifrost, while that changes was at SDK mpos part of code. They are both in same repo, but are different even than dependent things.

And c1694b6f824b02789942a16d1c4c2a916d7e4886 and 3ac9693d02d6cd2c2eaa7c039f7abc48114d863f seems like different to me.

**Comment by darakeon** [comment]
@guarilha , check the commit to see if it is ok the docs changes

**Comment by guarilha** [comment]
@darakeon sorry, makes sense. let's keep it this way. I'm going to review the docs changes now

**Pull request merged**